app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -10,17 +10,23 @@
 
     var trackingOptions = getOptionsFromCookie();
 
-    // Track initial pageview
-    this.trackPageview(null, null, trackingOptions);
-
-    // Begin error and print tracking
-    GOVUK.analyticsPlugins.error({filenameMustMatch: /gov\.uk/});
-    GOVUK.analyticsPlugins.printIntent();
-    GOVUK.analyticsPlugins.mailtoLinkTracker();
-    GOVUK.analyticsPlugins.externalLinkTracker();
-    GOVUK.analyticsPlugins.downloadLinkTracker({
-      selector: 'a[href*="/government/uploads"], a[href*="assets.publishing.service.gov.uk"]'
-    });
+    // We're setting the client ID inside a callback function because
+    // we don't have access to the client ID until GA returns a tracker object.
+    ga(function (tracker) {
```

Comment:
Sorry, I didn't see your review here. 

Yes, you're correct, we aren't interacting with it directly, which is why I've had to do this workaround in order to obtain the client id since it's not being set in static. It's being set in `govuk_frontend_toolkit` where we communicate with GA through 2 trackers: GOVUKTracker and GoogleAnalyticsUniversalTracker.

So we've had to wait for the tracker to be returned to us (in this case GoogleAnalyticsUniversalTracker) so that we could access the client id.